projects/habitat_ovmm/configs/agent/floorplanner_eval.yaml

```diff
@@ -17,7 +17,7 @@ ENVIRONMENT:
   min_depth: 0.0          # minimum depth for depth sensor (in metres)
   max_depth: 10.0         # maximum depth for depth sensor (in metres)
   num_receptacles: 21
-  goal_type: ovmm
+  category_map_file: data/datasets/floorplanner/rearrange/v3/val/cat_map.json
```

Comment:
before release we need to go through and change floorplanner to hssd (or just rearrange)